stapi_fastapi/__dev__.py

```diff
@@ -34,8 +34,19 @@ def validate_backend(cls, value: str):
 
 settings = DevSettings()
 backend = settings.backend_name()
+
+# Compose the router
+main_router = MainRouter()
+main_router.include_router(umbra_spotlight_router, prefix="/umbra_spotlight")
```

Comment:
I was thinking we would create a method on the `MainRouter` class that would take a product router as the sole input, as it should be able to determine the correct prefix for the `.include_router()` call based on the product router's product's `id` (if that chain of entities makes any sense... 🙄).

Reply:
Another aspect of adding a product router to the `MainRouter` is that the `MainRouter` instance needs to track all added product routers so it can iterate through them to build the list of products to return for the `GET /products` endpoint.